release-notes.md

```diff
@@ -1,5 +1,15 @@
 # Release notes
 
+## September 10, 2019
+
+The following updates were made in this release:
+
+**Android Analytics 1.2.0**
+
+* Adds two new pieces of data when reporting a crash: `previousosversion` and `previousappid`.
```

Comment:
Please reword the 1st bullet point to:
"Added the following new pieces of data when reporting a crash: 
* `previousosversion`
* `previousappid`.